index.html

```diff
@@ -212,6 +211,39 @@ <h1 class="about-section-h1">About Me</h1>
                 </ul>
             </div>
         </section>
+
+        <section class="form-section">
+            <h2 class="form-h2">I'm always<br>interested in hearing<br>about new projects,<br>so if you'd like to<br>chat please get in touch.</h2>
```

Comment:
- [ ] **OPTIONAL** I offer you to make your margin and padding the same with the template, so this designs were created by masters, and it is better to follow them. Look at the images below:

![image](https://user-images.githubusercontent.com/77872355/129790071-11579844-9f17-4575-a171-265ff932ae5f.png)
![image](https://user-images.githubusercontent.com/77872355/129790138-8201152a-84f7-4921-b33c-c20d30ecb183.png)